langs/it.tsx

```diff
@@ -1433,18 +1433,18 @@ const messages:IMessages = {
     h: "aLgG0qubuw0p0tGIUYKjgxecVsL0e3AKvQ5+W2JZg18=",
   },
   "Services": {
-    val: "Services",
+    val: "Servizi",
     translated: false,
     h: "Y5+GuNZXfFSynDsHvAY8cr0RqU+moaHZNmOk2eQrsNs=",
   },
-  "Set mapping": {
+  "Imposta mappatura": {
```

Comment:
Oops, this needs to be changed back to `"Set mapping"` and the line below should be `val: "Imposta mappatura",`